style.md

```diff
@@ -0,0 +1,128 @@
+# Style
+This document describes the style guide for the website and animations.
+
+# Animations
+## Timing
+Avoid running animations with non-standard units of time. Prefer using one of the following values:
+`0, 0.25, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 5...`
+When setting the pace of an animation, consider that animations may play in a format which is not easily
+pausable, and that many users prefer a slower format. 
+
+Animations should always finish with a delay equal to `END_DELAY`.
+
+## Colors
+Use the `Color` type and colors defined in `style`. Do not use colors from manim directly.
+
+## Building Animations
+Animations used on the live website should be built as `gif`s with medium (`-qm`) quality. The output file
+name (`-o [FILENAME]`) should be lowercase with underscores for spaces.
+```
+%%manim -v CRITICAL --format gif --disable_caching -qm -o boundary_constraints BoundaryConstraintScene
+```
+
+## Using Animations
+Animations should generally be displayed using the `figure` environment. 
```

Comment:
I must admit I don't know what this means. We should have some guides on how to prepare a document for the website.